src/Namotion.Reflection/Namotion.Reflection.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net462</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net462;net6.0</TargetFrameworks>
```

Comment:
do we even need .NET 6?

Reply:
I tried removing it from netstandard2.0 and IIRC it gave an error about not having the reference.

Reply:
This actually comes from Namotion.Reflection.Cecil and its need to call XmlDocs APIs with `dynamic` type to provide both Cecil types and standard CLR types. Is the Namotion.Reflection.Cecil library actually needed anymore?

Reply:
It would be great to have a version without dependency on Microsoft.CSharp or reference a more recent version as 4.3 brings a lot of other legacies. See example from our packages lock file
![image](https://github.com/RicoSuter/Namotion.Reflection/assets/23052247/5545592f-d41b-4292-ae94-8058ed9d0dce)